src/org/zaproxy/zap/extension/api/OptionsParamApi.java

```diff
@@ -34,12 +34,16 @@
 	private static final Logger LOGGER = Logger.getLogger(OptionsParamApi.class);
 
 	public static final String ENABLED = "api.enabled";
+	public static final String UI_ENABLED = "api.uienabled";
 	public static final String SECURE_ONLY = "api.secure";
 	public static final String API_KEY = "api.key";
 	private static final String DISABLE_KEY = "api.disablekey";
 	private static final String INC_ERROR_DETAILS = "api.incerrordetails";
 	private static final String AUTOFILL_KEY = "api.autofillkey";
 	private static final String ENABLE_JSONP = "api.enablejsonp";
+	private static final String NO_KEY_FOR_VIEWS = "api.nokeyforviews";

```

Comment:
This could apply to all endpoints (i.e. also "other" endpoints), to ensure that not data is leaked.